assets/operator-controller/01-customresourcedefinition-operators.operators.operatorframework.io.yml

```diff
@@ -0,0 +1,109 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.10.0
+  creationTimestamp: null
```

Comment:
Oh, we should bump upstream to use v0.11.4. I made a fix there that removes `creationTimestamp: null`, which kustomize v5 barfs on.

I'll work on this.

Reply:
Looks like there's a v0.12.0 now, so bumping all upstream repos to that:
- https://github.com/operator-framework/operator-controller/pull/281
- https://github.com/operator-framework/rukpak/pull/647
- https://github.com/operator-framework/catalogd/pull/105